src/api/firebase/firebase.api.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/no-duplicates */
```

Comment:
It shouldn't be here. Please import stuff in proper way.

Reply:
https://www.npmjs.com/package/firebase - please check docs at npm:
```js
// This import loads the firebase namespace along with all its type information.
import firebase from 'firebase/app';

// These imports load individual services into the firebase namespace.
import 'firebase/auth';
import 'firebase/database';
```
I don't know if we can resolve the ESLint issue in this case.

Reply:
Fine.